ompi/mpi/c/request_get_status_any.c.in

```diff
@@ -73,7 +73,7 @@ PROTOTYPE ERROR_CLASS request_get_status_any(INT count, REQUEST_CONST requests:c
         *flag = true;
         *indx = MPI_UNDEFINED;
         if (MPI_STATUS_IGNORE != status) {
-            OMPI_COPY_STATUS(status, ompi_status_empty, false);
+            OMPI_COPY_STATUS(status, ompi_status_empty, true);
```

Comment:
This is not what the standard mandates. Section 3.2.5 it states:
> ... message-passing calls do not modify the value of the error code field of
status variables. This field may be updated only by the functions in Section 3.7.5 that
return multiple statuses. The field is updated if and only if such function returns with an
error code of MPI_ERR_IN_STATUS.

In most of these cases the return is MPI_SUCCESS so the MPI_ERROR shall not be modified.